spk/debian-chroot/src/installer.sh

```diff
@@ -23,6 +23,10 @@ postinst ()
     # Link
     ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
 
+    # Set the permissions
+    chown -hR root:root ${SYNOPKG_PKGDEST}
+    chmod -R go-w ${SYNOPKG_PKGDEST}
```

Comment:
I thought #1226 adds the necessary code to be able to remove this part of this PR, doesn't it?